wgsl/index.bs

```diff
@@ -442,18 +548,22 @@ storage_class
 
 TODO: *The following is a stub*
 
-Variables in certain storage classes must have host-shareable store type with fully elaborated memory layout.
+Variables in certain storage classes must have a [=host-shareable=] [=store type=] with fully elaborated memory layout.
```

Comment:
would it be cleaner to say "Buffer variables must have a ..."?

Reply:
"buffer variable" is not yet a defined term at this point in the spec.

I can look at rewording it with more connective tissue.

Reply:
Can we elaborate `certain storage classes` into the storage class list `uniform and storage classes` or something?